include/coro/thread_pool.hpp

```diff
@@ -215,7 +218,11 @@ class thread_pool
     /// The configuration options.
     options m_opts;
     /// The background executor threads.
+#ifdef __cpp_lib_jthread
     std::vector<std::jthread> m_threads;
+#else
```

Comment:
I'm kind of thinking for easy of maintenance we just stick with std::thread and manually handle the stop, std::jthread is nice that it encapsulates it but I think the code will be cleaner and easier to maintain if we stick to one solution.  Unless if you think in the future emscripten will support std::jthread and we can bump the required version of that and just support std::jthread from that point moving forward? What do you think?

Reply:
I'm assuming that the emscripten folk would support std::jthreads at some point. I'm really surprised it's not working already as I would have thought its only a matter of recompiling the C++ standard library. It's up to you how you want to work it until they do support std::jthreads. I'm fine either way.

Reply:
Sorry for taking so long to get back to this, been insanely busy.

I think we'll keep it as is and once emscripten supports jthread we'll remove the std::thread solution.